client/blocks/edit-gravatar/test/index.jsx

```diff
@@ -1,33 +1,74 @@
 /**
  * @jest-environment jsdom
  */
-import { shallow } from 'enzyme';
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import { EditGravatar } from 'calypso/blocks/edit-gravatar';
 import ImageEditor from 'calypso/blocks/image-editor';
-import DropZone from 'calypso/components/drop-zone';
-import VerifyEmailDialog from 'calypso/components/email-verification/email-verification-dialog';
 import FilePicker from 'calypso/components/file-picker';
-import Gravatar from 'calypso/components/gravatar';
 import { AspectRatios } from 'calypso/state/editor/image-editor/constants';
-
-jest.mock( 'event', () => require( 'component-event' ), { virtual: true } );
-jest.mock( 'calypso/lib/oauth-token', () => ( {
-	getToken: () => 'bearerToken',
+import { renderWithProvider } from 'calypso/test-helpers/testing-library';
+
+jest.mock( 'calypso/components/email-verification/email-verification-dialog', () => () => (
+	<div data-testid="verify-email-dialog" />
+) );
+jest.mock( 'calypso/components/drop-zone', () => ( { children } ) => (
+	<div data-testid="dropzone">{ children }</div>
+) );
+jest.mock( 'calypso/blocks/image-editor', () =>
+	jest.fn( ( { onDone } ) => (
+		<div data-testid="image-editor">
+			<button data-testid="image-editor-done" onClick={ () => onDone() }>
+				Done
+			</button>
+		</div>
+	) )
+);
+jest.mock( 'calypso/components/gravatar', () => () => <div data-testid="gravatar" /> );
+jest.mock( 'calypso/components/file-picker', () =>
+	jest.fn( ( { onPick, children } ) => (
+		<div data-testid="file-picker">
+			{ children }
+			<input
+				data-testid="file-picker-input"
+				type="file"
+				onChange={ ( e ) => {
+					onPick( e.target.files );
+				} }
+			/>
+		</div>
+	) )
+);
+jest.mock( '@automattic/components', () => ( {
+	__esModule: true,
+	Dialog: ( { children } ) => <div>{ children }</div>,
+	Gridicon: () => <div>gridicon</div>,
+	Spinner: () => <div>spinner</div>,
+	Button: () => <div>button</div>,
 } ) );
```

Comment:
We should be able to remove these `@automattic/components` mocks by just doing something like this on every test:

```
import Modal from 'react-modal';

....

const { container } = render( <Something /> );
Modal.setAppElement( container );
```

Reply:
Good call! Removed those mocks via https://github.com/Automattic/wp-calypso/pull/66808/commits/766d522cffeba1958993c64271168e80b73ffd9